optimum/quanto/nn/qmodule.py

```diff
@@ -200,13 +201,15 @@ def from_module(
         activations: Optional[qtype] = None,
         optimizer: Optional[Optimizer] = None,
     ):
-        qmodule = cls.qcreate(module, weights, activations, optimizer)
+        with init_empty_weights():
```

Comment:
init_empty_weights() is called since it prevents weight initialization from happening, which was the main cause of the slow performance of quantize/requantize